account_invoice_date/README.rst

```diff
@@ -0,0 +1,6 @@
+====================
+Account Invoice Date
+====================
+
+When an invoice is validated from Draft to Open automatically, the invoice date
+is set to current date.If an invoice is validated from Draft to Open manually, a window is displayed to query the invoice date from the user.
```

Comment:
```suggestion
is set to current date. If an invoice is validated from Draft to Open manually, a window is displayed to query the invoice date from the user.
```

Can you put this in the first comment of the PR? this is a new module. People should know what is this about without looking at the code. Thanks.